CovertInfiltration/Localization/XComGame.int

```diff
@@ -351,18 +351,14 @@ ConsoleObjectiveTextPools[0]=
 ConsoleObjectiveTextPools[1]=
 
 [GatecrasherCI X2MissionTemplate]
-DisplayName="Operation Gatecrasher"
-Briefing="ADVENT has brought a pair of dissident academics to this facility for 're-education'. These two could be of extreme value to the Resistance. Ensure they both make it out of here alive."
+DisplayName="Rescue VIPs from ADVENT Compound"
+Briefing="Rescue VIPs from ADVENT Compound"
```

Comment:
In line with "Gatecrasher", could this be something like "break into advent compound and rescue VIPs"?